### DISCUSSION TITLE
youtube视频增加双语lrc文件制作

### DISCUSSION THREAD

**Discussion by YesHub** [immersive-translate/immersive-translate#712] [Ideas]
### 新功能 | New Feature

1.使用视频字幕制作lrc文件
2.可以选择不同语言制作对应双语lrc
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.觉得这对学习英语非常有用
2.有了双语lrc文件可以不必每次都需要进入youtube看视频，非常方便。
3.

**Comment by theowenyoung** [comment]
有没有试过已有的字幕文件翻译功能？

<img width="308" alt="图片" src="https://github.com/immersive-translate/immersive-translate/assets/62473795/375e5107-0532-4708-82ec-d0dfd435a365">
